lb-haproxy-study/README.md

```diff
@@ -0,0 +1,49 @@
+# Configure HAProxy on control plane as a daemonset on top of Kubernetes or as a service on plain Linux
+
+## HAPROXY as daemonset or service
+* Assumption for all options is that needed **cloud-init** can be created from userdata given in Kubernetes CRs
+* Goal is to describe control plane load balancing options without cluster external load balancer
```

Comment:
cluster external?

Reply:
meaning external LB, I see this clear enough.